### PR TITLE
Fixed #3, Traversal of the space of train/test splits.

### LINKED ISSUE #3: [Outreachy applications] Traversal of the space of train/test splits
Given a classification model, we want to investigate how much the performance score computed on the test set depends on the choice of train/test split proportion. Eg. how would our performance estimate change if we used a 60/40 split rather than 80/20?

Write a function that takes a scikit-learn estimator and a dataset, and computes an evaluation metric over a grid of train/test split proportions from 0 to 100%. To assess variability, for each split proportion it should resplit and recompute the metric multiple times. It should output a table of splits with multiple metric values per split.

### PR DESCRIPTION
**Pull request by Bolaji61**
Here, I have been able to create a `traversal()` function that takes estimator and dataset as parameters and returns a result of different test sizes and their corresponding accuracy score based on the Logistic Regression estimator used.

However, I do not yet understand what is meant by "a table of splits with multiple metric values per split" as mentioned in issue #3 , I'll appreciate if this is further clarified.

### CODE REVIEW

**Comment by Bolaji61** [comment]
I now understand that multiple metrics includes F1-score, Recall, Precision, Classification report, therefore I have added these metrics to each train/test split. However I still have a challenge with finding out a way to improve the view of these metrics.

**Comment by Bolaji61** [comment]
I have fixed issue #3, by investigating and explaining on how much the performance metrics/scores depend on the choice of train/test split proportions using Logistic Regression model
